components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -58,6 +62,14 @@ - (void)setUpLayoutDirection {
   self.layoutDirection = self.mdf_effectiveUserInterfaceLayoutDirection;
 }
 
+- (void)setUpClearButton {
+  self.clearButton = [[MDCContainedInputViewClearButton alloc] init];
+  [self.clearButton addTarget:self
+                       action:@selector(clearButtonPressed:)
+             forControlEvents:UIControlEventTouchUpInside];
+  [self addSubview:self.clearButton];
+}
```

Comment:
How does this clear button interact with the existing clear button on the text field? As-is, it looks like both could co-exist.

Reply:
I made it so they don't conflict with one another 👍